README-template.md

```diff
@@ -78,7 +78,7 @@ To add a dependency on one of the connectors using Maven, use the following:
     <dependency>
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>gcs-connector</artifactId>
-      <version>${next-gcs-connector-release-tag}</version>
+      <version>hadoop2-${next-gcs-connector-release-tag}</version>
```

Comment:
We also publish for hadoop3 right?

Reply:
Yes, we publish for both hadoop2 and hadoop3 from this branch.